devito/finite_differences/finite_difference.py

```diff
@@ -277,6 +324,13 @@ def staggered_diff(expr, deriv_order, dim, fd_order, stagger=centered):
     :param stagger: (Optional) shift for the FD, `left`, `right` or `centered`.
     """
 
+    # Check if we need symbolic fd weights (not implemented)
```

Comment:
I've seen this code already (above I think). Perhaps we need a decorator?

Reply:
Yeah, this repetition can and should be avoided.